backend/app/environments/schema.py

```diff
@@ -18,27 +17,43 @@ class GetEnvironment(Environment):
 
 
 class _AWSS3Config(ORMModel):
-    account_id: int
-    name: str
+    identifier: str
+    bucket_name: str
     arn: str
-    kms: str
+    kms_key: str
+    is_default: bool
 
 
-class Config(RootModel[Dict[str, _AWSS3Config]]):
-    pass
+class _AWSGlueConfig(ORMModel):
+    identifier: str
+    database_name: str
+    bucket_name: str
+    s3_path: str
 
 
 class EnvPlatformServiceConfig(ORMModel):
-    config: Config
+    config: list[_AWSS3Config | _AWSGlueConfig]
 
     @field_validator("config", mode="before")
     @classmethod
-    def parse_settings(cls, v: str | dict) -> dict:
+    def parse_settings(cls, v: str | list) -> list:
         if isinstance(v, str):
             return json.loads(v)
         return v
 
 
-class CreateConfigSchema(EnvPlatformServiceConfig):
-    platform_id: UUID
-    service_id: UUID
+class _AWSConfig(ORMModel):
+    account_id: str
+    region: str
+    can_read_from: list[str]
+
+
+class EnvPlatformConfig(ORMModel):
```

Comment:
I would also split up the platform and platform service configs. This feels like a god file now

Reply:
Why do you mean under `platform` and `platform service configs`?